sdks/java/io/hadoop/input-format/src/main/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIO.java

```diff
@@ -389,6 +394,7 @@ public void populateDisplayData(DisplayData.Builder builder) {
     private long boundedSourceEstimatedSize = 0;
```

Comment:
Was looking at this issue, added a log at info level in populateDisplayData(), and I never saw the log in worker logs, seems like this method was never called.